website/src/components/SequenceDetailsPage/getDataTableData.ts

```diff
@@ -71,7 +79,9 @@ export function getDataTableData(listTableDataEntries: TableDataEntry[]): DataTa
             result.topmatter.authors = entry.value
                 .toString()
                 .split(';')
-                .map((x) => x.trim());
+                .map((x) => x.trim())
+                .filter((x) => x.length > 0)
```

Comment:
```suggestion
                .filter((x) => x.trim().length > 0)
```

since we are also trimming in `formatAuthorName`